src/transition-jira-issues.ps1

```diff
@@ -184,32 +186,32 @@ try {
     }
 
     If ($identifiedIssueKeys.Length -eq 0 -And !$FailOnTransitionFailure) {
-        Write-Output "::warning title=$MESSAGE_TITLE::No issues were successfully transitioned. See job logs for details and action $env:GITHUB_ACTION_URL for additional help."
+        If ($CreateWarningNotices) { Write-Output "::warning title=$MESSAGE_TITLE::No issues were successfully transitioned. See job logs for details and action $env:GITHUB_ACTION_URL for additional help." }
```

Comment:
Same question here about whether or not we should have some kind of output. Also for the rest of the spots that checks were added (I just don't want to have to leave a bunch of comments since the answers are all going to be the same).

Reply:
Same as my response above.  When this warning notice is disabled, is up to the caller to look at the output to determine what they want to do or notify.

Reply:
It is similar to the other actions in `im-open` that have a `fail-on-error` input.  When disabled, its up to the caller to properly handle failures.

Reply:
The `fail-on-error` doesn't usually hide output, just fails the action. That's the main thing I'm worried about. It definitely makes sense to not show an in-your-face warning for each problem, especially when you're running this action multiple times, I'm only worried that you wouldn't know why there was a problem. If you feel that's easy enough to glean from the outputs the action has, then my fear is alleviated.

Reply:
If you look at the action log, you will see the warning.  And like you mentioned, the action outputs will also indicate what the warnings are also indicating.  It just doesn't add it as a warning notice.

Reply:
Example: 
<img width="1630" alt="image" src="https://github.com/im-open/transition-jira-issues/assets/8923247/80c0cca9-e24e-462b-b261-a45d7b39a051">

https://github.com/im-enrollment/ifp-connector-infrastructure/actions/runs/5381639130/jobs/9766127274